env/env_hdfs.cc

```diff
@@ -238,6 +238,11 @@ class HdfsWritableFile: public WritableFile {
     return Status::OK();
   }
 
+  virtual Status Append(const Slice& data,
```

Comment:
The default implementation is already defined in the base class, so this shouldn't be required. Can't it be solved by adding ```using WritableFile::Append```?